app/forms/password_form.rb

```diff
@@ -10,10 +10,12 @@ def initialize(user)
   end
 
   def submit(params)
-    self.password = params[:password]
+    submitted_password = params[:password]
+
+    self.password = submitted_password
 
     if valid? && user_valid?
-      @user.password = params[:password]
+      @user.password = submitted_password
```

Comment:
I don't think it looks worse. I think `submitted_password` makes it clear where the password came from. I don't have strong feelings either way.